src/config.nim

```diff
@@ -153,6 +155,11 @@ proc readSettings*(path: string): Settings =
             else:
               result.titanPassRequired = true
           of "titanuploadlimit": result.titanUploadLimit = e.value.parseInt
+          of "titanredirect":
+            if e.value == "false":
+              result.titanRedirect = false
```

Comment:
Consider using [parseBool](https://nim-lang.org/docs/strutils.html#parseBool%2Cstring) instead

Reply:
Ah thanks -- I really need to familiarize myself with the standard library!